components/html-block/demo/html-block.html

```diff
@@ -9,6 +9,61 @@
 		<script type="module">
 			import '../../demo/demo-page.js';
 			import '../html-block.js';
+			import { provideInstance } from '../../../mixins/provider-mixin.js';
+
+			let profileCardAccess;
+			const hasProfileCardAccess = () => {
+				if (profileCardAccess) return profileCardAccess;
+
+				profileCardAccess = new Promise(resolve => {
+
+					resolve(true); // temp
+
+					if (!D2L.LP) {
+						resolve(false);
+						return;
+					}
+
+					D2L.LP.Web.UI.Rpc.Connect(
+						D2L.LP.Web.UI.Rpc.Verbs.GET,
+						new D2L.LP.Web.Http.UrlLocation('/d2l/lp/profilecardsaccess/AccessCheck'),
+						null,
+						{ success: response => resolve(response) }
+					);
+				});
+
+				return profileCardAccess;
+			};
+
+			provideInstance(document, 'html-block-renderers', [
+				async elem => {
+
+					const mentions = elem.querySelectorAll('[data-mentions-id]');
+					if (mentions.length === 0) return elem;
+
+					const hasAccess = await hasProfileCardAccess();
```

Comment:
Currently, this call would block rendering of the rest of the user-authored HTML.  Perhaps what would actually be preferred is if this would render a web component that can render itself asynchronously and conditionally based on the access API result, so this code does not have to `await` the API.  In this case this could happen at most once per page.